content/docs/pipelines/sdk/client_usage.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Use Kubeflow Pipelines API Client"
+description = "KFP SDK version >= 0.2.5 is required"
+weight = 30
++++
+
+This page includes a quick example on how to create a pipeline and a pipeline version using KFP API client.
+
+
+## Before you start
+
+* Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/). It is recommended to always use the latest KFP SDK.
```

Comment:
Recommend telling them they need to upgrade `kfp-server-api` to 0.2.5 or above.
and use `pip list | grep kfp` for checking currently installed version.

Reply:
done